styles.css

```diff
@@ -0,0 +1,585 @@
+
+* {
+  scrollbar-width: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none;
+}
+
+.check,
+.check * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+.cw,
+.cw * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+#header {
+  width: 100vw;
+  left: 0;
+  z-index: 10;
+  background-color: white;
+}
+
+#header a {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+#header a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#header-nav-bar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 37%;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
```

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓

Reply:
Thanks for this suggestion! I have added some classes to reduce code like recommended!

Reply:
- [ ] This issue is well resolved :heavy_check_mark: